src/test/rustdoc-gui/impl-default-expansion.goml

```diff
@@ -0,0 +1,4 @@
+// This test ensures that the impl blocks are open by default.
+goto: file://|DOC_PATH|/index.html
+wait-for: 5000
+assert: ("#main > details.implementors-toggle", "open", "")
```

Comment:
Can you add a test that intentionally fails to make sure this runs at all?

Reply:
They are not run, that's for sure. Otherwise, it'd have failed a long time ago because of `nojs-attr-pos.goml`.

Reply:
Ok. In the meantime, can you run it locally and make sure it passes?

Reply:
This version didn't because I forgot to commit the one with the correct URL (`struct.Foo.html` and not `index.html`). But otherwise it works as expected.

Reply:
Can you fix it to have the correct URL? I don't see the point of adding a test we know will fail.

While you're add it, can you update the other test to pass instead of removing it altogether?

Reply:
Look at my [comment below](https://github.com/rust-lang/rust/pull/84552#issuecomment-826328066).

Reply:
> While you're at it, can you update the other test to pass instead of removing it altogether?

The test which fails checked for attributes toggles. Since they don't have toggles anymore, there is no point of keeping the test.

Reply:
> The test which fails checked for attributes toggles. Since they don't have toggles anymore, there is no point of keeping the test.

Can you instead test that it doesn't have toggles?

Reply:
Would it be useful to check that something was removed? For example, if we put back toggles on attributes with a different DOM, the test would still be happy. Which is why it's simpler to enforce what exists over what doesn't. (Not sure if that makes sense said like this ><)

Reply:
I agree with @GuillaumeGomez. When removing "code that does X," it's often tempting to add a test for "doesn't do X anymore." But the problem is there are a zillion ways it could wind up doing X again, and it's impossible to cover all of them (or even a reasonable fraction). It winds up adding to the test running time, and the burden of maintaining tests, without a corresponding benefit.